ui/pages/settings/settings.container.js

```diff
@@ -144,12 +145,19 @@ const mapStateToProps = (state, ownProps) => {
   const snapSettingsTitle =
     isSnapSettingsRoute && snapNameGetter(decodeSnapIdFromPathname(pathname));
 
+  const { subscriptions } = getUserSubscriptions(state);
+  // get shield subscription
+  const shieldSubscription = subscriptions.find((subscription) =>
+    subscription.products.some((product) => product.name === 'shield'),
```

Comment:
isn't the product name a constant? if not, we should make it one and it should be in core package
@tuna1207

Reply:
can do in the next pr along with core update